tests/topotests/bgp_multi_vrf_topo2/test_bgp_multi_vrf_topo2.py

```diff
@@ -2011,6 +2033,1783 @@ def test_vrf_route_leaking_next_hop_interface_flapping_p1(request):
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        },
+                    },
+                },
+                {
+                    "local_as": "200",
+                    "vrf": "BLUE_B",
+                    "address_family": {
+                        "ipv4": {
+                            "unicast": {
+                                "neighbor": {
+                                    "blue2": {
+                                        "dest_link": {
+                                            "r3-link2": {
+                                                "keepalivetimer": KEEPALIVETIMER,
+                                                "holddowntimer": HOLDDOWNTIMER,
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        },
+                        "ipv6": {
+                            "unicast": {
+                                "neighbor": {
+                                    "blue2": {
+                                        "dest_link": {
+                                            "r3-link2": {
+                                                "keepalivetimer": KEEPALIVETIMER,
+                                                "holddowntimer": HOLDDOWNTIMER,
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        },
+                    },
+                },
+            ]
+        }
+    }
+
+    result = create_router_bgp(tgen, topo_modify, input_dict_4)
+    assert result is True, "Testcase {} :Failed \n Error: {}".format(tc_name, result)
+
+    for addr_type in ADDR_TYPES:
+        clear_bgp(tgen, addr_type, "r3", vrf=["PINK_A", "RED_B", "GREY_A", "BLUE_B"])
+
+    step(
+        "After deleting VRFs ipv6 addresses wil be deleted from kernel "
+        " Adding back ipv6 addresses"
+    )
+
+    dut = "r3"
+    vrfs = ["GREY_A", "PINK_A"]
+
+    for vrf in vrfs:
+        for c_link, c_data in topo_modify["routers"][dut]["links"].items():
+            if c_data["vrf"] != vrf:
+                continue
+
+            intf_name = c_data["interface"]
+            intf_ipv6 = c_data["ipv6"]
+
+            create_interface_in_kernel(
+                tgen, dut, intf_name, intf_ipv6, vrf, create=False
+            )
+
+    step("Waiting for holdowntimer+1 sec..")
+    sleep(HOLDDOWNTIMER + 1)
+
+    step(
+        "Advertised prefixes should appear again in respective VRF"
+        " table on routers RED_2 and BLUE_2. For verification use same"
+        " commands as mention in step-3.."
```

Comment:
what is the step-3 here? how to find it out?

Reply:
I have updated verification step.